src/server/api/epicRouter.ts

```diff
@@ -74,7 +74,13 @@ export const buildEpicRouter = (
 
             return [...hardcodedTests, ...articleEpicTests.get()];
         } catch (err) {
-            logWarn(`Error getting article epic tests: ${err}`);
```

Comment:
elsewhere in this repo we just convert the error to a String, e.g. [here](https://github.com/guardian/support-dotcom-components/blob/main/src/server/lib/promotions/promotions.ts#L79).
How about replacing this with that?
`logWarn(`Error getting article epic tests: ${String(err)}`);`